.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Welcome! Thanks for your issue. In order for us to investigate your issue, it really helps to make sure we have the information we need. Please fill out the form sections below so we can help you as promptly as we can. Thanks! -->
+
+## Tell us about your runtime:
+
+* **OS**:
+* **QUnit version**:
+* **How are you running QUnit? (e.g., browser, Grunt)**:
+
+<!--
+Please be specific about your runtime.
+
+* If you're running via browser, please include browser and version.
+* If you're running via a Node.js integration such as Grunt, please include your Node.js version and the name and version of your integration.
+
+Please include anything else that you believe may be helpful for us to know.
+-->
+
+## What are you trying to do?
+
+Code that reproduces the problem:
```

Comment:
We should likely add a comment to the effect of "Please provide an exact reproduction of your issue if possible. If not, then please provide detailed information about your setup and what you are trying to do."

This is because sometimes developers can't share the actual code (e.g., when working on private projects) and may not have time to get a minimal reproduction setup before filing the issue.